### ISSUE TITLE
create daps-jing for debian

### ISSUE THREAD

**Issue by tbazant** [openSUSE/daps#336]
see the template:

```
#!/bin/sh

# Include the wrappers utility script
. /usr/lib/java-wrappers/java-wrappers.sh

# We prefer to use openjdk or Sun's java if available
find_java_runtime openjdk sun  || find_java_runtime 

# Set dfault (and local) classpath
JAVA_CLASSPATH="/usr/share/java"

if [ "$1" = "classpath" ]
then
  JAVA_CLASSPATH="$2"
  shift 2
fi

export CLASSPATH=$JAVA_CLASSPATH

JAVA_ARGS="-Dorg.apache.xerces.xni.parser.XMLParserConfiguration=org.apache.xerces.parsers.XIncludeParserConfiguration"

DEBUG_WRAPPER=1

#find_jars jing xerces-j2 xml-commons-apis saxon6 xml-commons-resolver 
find_jars jing xercesImpl xmlParserAPIs xml-apis saxon xml-resolver avalon-framework batik-all 

run_java com.thaiopensource.relaxng.util.Driver "$@"
```

**Comment by tbazant** [comment]
the bug is already fixed in unstable, see http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=807678

**Comment by fsundermeyer** [comment]
Fixed with 80af9d430beb09100498d570a18565add2d46eda
